content/hi/docs/reference/glossary/disruption.md

```diff
@@ -0,0 +1,25 @@
+---
+title: व्यवधान(Disruption)
+id: व्यवधान(disruption)
+date: 2019-09-10
+full_link: /docs/concepts/workloads/pods/disruptions/
+short_description: >
+  एक घटना जो पॉड (ओं) को सेवा से बाहर कर देती है
+aka:
+tags:
+- fundamental
+---
+ व्यवधान ऐसी घटनाएँ हैं जो एक या अधिक की ओर ले जाती हैं
+{{< glossary_tooltip term_id="pod" text="Pods" >}} सेवा से बाहर हो रहा है।
+कार्यभार संसाधनों के लिए एक व्यवधान के परिणाम होते हैं, जैसे
+{{< glossary_tooltip term_id="तैनाती" >}}, जो प्रभावितों पर निर्भर करता है
+फली।
+
+<!--more-->
+
+यदि आप, क्लस्टर ऑपरेटर के रूप में, किसी एप्लिकेशन से संबंधित पॉड को नष्ट करते हैं,
+कुबेरनेट्स का कहना है कि एक _स्वैच्छिक व्यवधान_। यदि कोई पॉड ऑफ़लाइन हो जाता है
+एक नोड विफलता, या एक व्यापक विफलता क्षेत्र को प्रभावित करने वाले आउटेज के कारण,
+कुबेरनेट्स का कहना है कि एक _अनैच्छिक व्यवधान_।
+
+अधिक जानकारी के लिए [व्यवधान] (/docs/concepts/workloads/pods/disruptions/ देखें।
```

Comment:
```suggestion
अधिक जानकारी के लिए [व्यवधान](/docs/concepts/workloads/pods/disruptions/) देखें।
```

Reply:
Please address this comment @surbhiahuja